.github/workflows/ci-pull-requests.yml

```diff
@@ -3,6 +3,20 @@ name: Test Pull Requests
 on: [pull_request]
 
 jobs:
+  check-commit-messages:
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      contents: read
+    steps:
+      # https://www.notion.so/n8n/Release-Process-fce65faea3d5403a85210f7e7a60d0f8
+      - uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(feat|fix|perf|test|docs|refactor|build|ci)'
```

Comment:
should the pattern be more specific to match this expected pattern instead `<type>(<scope>): <subject>`

Reply:
That's a good point.

If we'd also want to soon lint commit messages in a pre-commit hook (e.g. with [commitlint](https://github.com/conventional-changelog/commitlint#what-is-commitlint)'s angular rules), it would be neat to extract both patterns from the same configuration to make maintenance easier.

Maybe if we'd factor in the optional scope for now as @mutdmour  proposed, and we can later look into making it fancier?

Reply:
yeh I'll definitely look into this, the regex is still a WIP and I was considering running the action multiple times with simpler regex's although I'm not sure I can :/ but I'll include scope checking in some way.

Reply:
Our documented release process doesn't require a scope, and unfortunately the [github action](https://github.com/marketplace/actions/semantic-pull-request) we're using doesn't make it easy to pattern match scopes (without dropping the entire implementing to a big regex). It won't fail if you do add a scope, but it won't check the scope is in the correct format. I propose we just try this out for now and see if it's adding value before adding stricter scope checking.